test/test_dtype_alu.py

```diff
@@ -19,6 +19,7 @@
 dtypes_float = (dtypes.float32, dtypes.float16)
 dtypes_int = (dtypes.int8, dtypes.int16, dtypes.int32, dtypes.int64, dtypes.uint8, dtypes.uint16, dtypes.uint32, dtypes.uint64)
 dtypes_bool = (dtypes.bool,)
+#binary_operations = [operator.add, operator.mul, operator.lt, operator.eq]
```

Comment:
Why is this here? This PR needs more cleanup, please read the diff until you think it's final.